Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/compositeeditor/BitFieldEditorDialog.java

```diff
@@ -211,7 +211,8 @@ private static String getCompositeType(Composite composite) {
 		if (composite.isInternallyAligned()) {
 			throw new IllegalArgumentException("Aligned use not supported");
 		}
-		String alignmentMode = composite.isInternallyAligned() ? "Aligned" : "Unaligned";
+		composite.isInternallyAligned();
+		String alignmentMode = "Unaligned";
```

Comment:
Actually this change should be correct. `composite.isInternallyAligned();` will always be executed, so this shouldn't have side-effects and it will always return "false" (because from my understanding `composite.isInternallyAligned();` should not change it's value by being invoked a certain amount of times), so alignmentMode will always be "Unaligned", because if `composite.isInternallyAligned();` was true, an exception would have been thrown before (in line 211-213)